### ISSUE TITLE
Breaks custom protocol

### ISSUE THREAD

**Issue by adam-lynch** [jakubpawlowicz/clean-css#608]
See https://github.com/murphydanger/gulp-minify-css/issues/107

**Comment by adam-lynch** [comment]
Hmm... :suspect: 

- gulp-minify-css@1.1.0 depends on ^3.1.5
- gulp-minify-css@1.1.1 depends on ^3.2.6

**Comment by shinnn** [comment]
@adam-lynch There is almost no difference between `^3.1.5` and `^3.2.6` since both installs clean-css v3.x. https://nodesource.com/blog/semver-tilde-and-caret

**Comment by jakubpawlowicz** [comment]
I don't think clean-css ever supported custom protocols, but can take a look into it.

**Comment by adam-lynch** [comment]
@jakubpawlowicz :+1: but hold on...  https://github.com/murphydanger/gulp-minify-css/issues/107#issuecomment-113159594

**Comment by shinnn** [comment]
It seems that clean-css breaks custom protocols only when the file path is provided.

### Setup

```
$ npm install clean-css@3.3.3
> clean-css@3.3.3 node_modules/clean-css
> ├── commander@2.8.1 (graceful-readlink@1.0.1)
> └── source-map@0.4.2 (amdefine@0.1.1)
$ node -v
> 2.3.0
```

### Test case 1: Without providing CSS file path :o:

#### Script

```js
'use strict';

var CleanCSS = require('clean-css');

var styles = new CleanCSS().minify('div{background:url(app://abc.png)}').styles;

console.log(styles);
```

#### Output

```css
div{background:url(app://abc.png)}
```

### Test case 2: Providing CSS file path :x:

#### Script

```js
'use strict';

var CleanCSS = require('clean-css');

var styles = new CleanCSS().minify({
	'styles.css': {
		styles: 'div{background:url(app://abc.png)}'
	}
}).styles;

console.log(styles);
```

#### Output

```css
div{background:url(app:/abc.png)}
```

**Comment by jakubpawlowicz** [comment]
@shinnn thanks for an (usual) outstanding bug report! The fix is coming.

**Comment by jakubpawlowicz** [comment]
Fixed in 3.3.5.

**Comment by adam-lynch** [comment]
:+1:
